charts/rqlite/templates/networkpolicy.yaml

```diff
@@ -0,0 +1,24 @@
+{{- if .Values.networkPolicy }}
+{{- if .Values.networkPolicy.enabled }}
+{{- $name := tpl (include "rqlite.fullname" .) $ -}}
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: {{ $name }}
+spec:
+  podSelector:
+    matchLabels:
+      {{- include "rqlite.selectorLabels" . | nindent 6 }}
+  ingress:
+    # Client-facing port, allow from all
```

Comment:
Client-facing port, runs HTTP(S), allow from all -- suggestion.

Reply:
In other words, I suggest you mention that the client-facing port is a HTTP server.

Reply:
Ack